docs/dev_guide/diagnostic_ref/dpct1032.rst

```diff
@@ -31,3 +31,37 @@ Suggestions to Fix
 ------------------
 
 You may need to adjust the code.
+
+For example, this original CUDA\* code:
+
+.. code-block:: cpp
+   :linenos:
+
+    __global__ void kernel(curandStateXORWOW_t rng) {
+      ...
+    }
+
+results in the following migrated SYCL code:

```

Comment:
Add "\*" to SYCL - not yet attributed on page. Need to attribute first use only.

Reply:
done

Reply:
done